ArduSub/motors.cpp

```diff
@@ -60,7 +60,7 @@ bool Sub::verify_motor_test()
 
     // Require at least 2 Hz incoming do_set_motor requests
     if (AP_HAL::millis() > last_do_motor_test_ms + 500) {
-        gcs().send_text(MAV_SEVERITY_WARNING, "Motor test timed out!");
+        gcs().send_text(MAV_SEVERITY_INFO, "Motor test timed out!");
```

Comment:
```suggestion
        gcs().send_text(MAV_SEVERITY_NOTICE, "Motor test timed out!");
```

@jaylthegreat Can you try MAV_SEVERITY_NOTICE https://mavlink.io/en/messages/common.html#MAV_SEVERITY_NOTICE?

Reply:
notice will also say it out loud. The code in the bluerobotics build of QGC has a line that basically says that if it's of MAV_SEVERITY_NOTICE or higher to read it out loud

Reply:
We ditched that branch in january.

We/you should be using mavlink/qgroundcontrol: https://github.com/mavlink/qgroundcontrol.

Reply:
https://github.com/mavlink/qgroundcontrol/blob/master/src/Vehicle/Vehicle.cc#L939

same applies